polus-precompute-slide-plugin/src/utils.py

```diff
@@ -19,6 +20,131 @@
 # Chunk Scale
 CHUNK_SIZE = 1024
 
+def image_generator(image):
+    mesh = image.ravel()
+    ids = [int(i) for i in np.unique(mesh[:])]
+    return ids
+
+def segmentinfo(encoder,idlabels,out_dir):
+    """ This function creates the info file needed to segment the image """
```

Comment:
No segmentation occurs in this plugin. I think you mean `needed to produce a Neuroglance segmentation.`